dw/customer/Customer.js

```diff
@@ -2,6 +2,7 @@ class Customer {
     constructor(profile) {
         this.ID = 'ID';
         this.profile = profile || new (require('./Profile'))();
+        this.AuthenticationStatus = require('./AuthenticationStatus');
```

Comment:
Not sure if this should be an instance and whether to pass `this` as a parameter?